### PR TITLE
Create mumble.sono.json

### PR DESCRIPTION
**Pull request by getSono**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by 21Z** [comment]
checks failing

File: domains/sono.json [hunk 1]
```diff
@@ -5,12 +5,13 @@
     },
     "record": {
         "MX": [
-            "mx01.mail.icloud.com",
-            "mx02.mail.icloud.com"
+            "mx1.improvmx.com",
+            "mx2.improvmx.com."
         ],
         "TXT": [
```
Comment (21Z) [inline domains/sono.json:11]:
```suggestion
        "TXT": "v=spf1 include:spf.improvmx.com ~all",
```

File: domains/sono.json [hunk 2]
```diff
@@ -5,12 +5,13 @@
...
-            "apple-domain=PMmXaZI06t5XN6Dv",
-            "v=spf1 include:icloud.com ~all"
-        ]
+            "v=spf1 include:spf.improvmx.com ~all"
+        ],
+
```
Comment (21Z) [inline domains/sono.json:14]:
```suggestion
```

**Review by 21Z — CHANGES_REQUESTED** [review]

File: domains/sono.json [hunk 3]
```diff
@@ -5,12 +5,10 @@
...
-            "mx02.mail.icloud.com"
+            "mx1.improvmx.com",
+            "mx2.improvmx.com."
         ],
-        "TXT": [
-            "apple-domain=PMmXaZI06t5XN6Dv",
-            "v=spf1 include:icloud.com ~all"
-        ]
+        "TXT": "v=spf1 include:spf.improvmx.com ~all",
+        "URL": "getsono.is-a.dev"
```
Comment (CuteDog5695) [inline domains/sono.json:12]:
```suggestion
        "URL": "https://getsono.is-a.dev"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/sono.json (hunk 3 above)
Comment (21Z) [inline domains/sono.json:12]: Missing protocol
```suggestion
        "URL": "https://getsono.is-a.dev"
```

**Review by 21Z — APPROVED** [review]

**Comment by 21Z** [comment]
watrado

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**